sql/core/src/main/scala/org/apache/spark/sql/execution/Columnar.scala

```diff
@@ -264,12 +264,12 @@ private object RowToColumnConverter {
       case DoubleType => DoubleConverter
       case StringType => StringConverter
       case CalendarIntervalType => CalendarConverter
-      case at: ArrayType => new ArrayConverter(getConverterForType(at.elementType, nullable))
+      case at: ArrayType => ArrayConverter(getConverterForType(at.elementType, at.containsNull))
```

Comment:
Ah, I see. Nice catch.

Reply:
Shouldn't it be `nullable || at.containsNull`?